dts/bindings/mdio/zephyr,mdio-i2c.yaml

```diff
@@ -0,0 +1,10 @@
+# Copyright The Zephyr Project Contributors
+# SPDX-License-Identifier: Apache-2.0
+
+description: Zephyr MDIO bitbang driver
```

Comment:
maybe it should be clarified how this binding is different from an I2C based ethernet phy